examples/deno-server/server.ts

```diff
@@ -0,0 +1,27 @@
+import { verifyKey } from "https://unpkg.com/@unkey/api@0.12.0/dist/index.mjs";
```

Comment:
Can you switch this too: `import { verifyKey } from "https://unpkg.com/@unkey/api@latest/dist/index.mjs";` 

So that it's not pinned to a specific API version.